code/procs/jobprocs.dm

```diff
@@ -306,6 +306,19 @@
 		picks = FindOccupationCandidates(staff,JOB.name,3)
 	return picks
 
+/proc/pilotSpawnVerif(var/turf/checkedTurf)				//Just roll with it.
```

Comment:
```suggestion
/proc/pilotSpawnVerif(turf/checkedTurf)				//Just roll with it.
```
Mostly just a stylistic choice but the `var/` is unnecessary.

Reply:
Actually nvm, I'd probably delete this whole proc - see comment below.

Reply:
Was allowed to keep the old code.

Reply:
Done.